cli/pcluster/cfnconfig.py

```diff
@@ -290,7 +290,7 @@ def __init__(self, args):
                         config_sanity.check_resource(self.region, self.aws_access_key_id, self.aws_secret_access_key,
                                                      'URL', self.template_url)
                 except configparser.NoOptionError:
-                    if self.region == 'us-gov-west-1':
+                    if self.region.startswith('us-gov'):
```

Comment:
if for us-gov the link "https://s3.%s.amazonaws.com/" works too, we could remove the elif and have
```
if region == 'us-east-1'
   https://s3.amazonaws.com/ ... 
else
   https://s3.%s.amazonaws.com/ ...
```
The same apply in the get_cookbook_url function